### ISSUE TITLE
Not Logging to database with ERROR

### ISSUE THREAD

**Issue by JuanAdamuz** [danielme85/laravel-log-to-db#11]
LOG:
`[2019-10-11 09:25:44] debug.DEBUG: This is an test DEBUG log event  
[2019-10-11 09:25:45] debug.ERROR: Argument 1 passed to danielme85\LaravelLogToDB\Models\DBLog::setDatetimeAttribute() must be an instance of Monolog\DateTimeImmutable, instance of DateTime given, called in \vendor\laravel\framework\src\Illuminate\Database\Eloquent\Concerns\HasAttributes.php on line 615 {"userId":9,"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Argument 1 passed to danielme85\\LaravelLogToDB\\Models\\DBLog::setDatetimeAttribute() must be an instance of Monolog\\DateTimeImmutable, instance of DateTime given, called in \\vendor\\laravel\\framework\\src\\Illuminate\\Database\\Eloquent\\Concerns\\HasAttributes.php on line 615 at \\vendor\\danielme85\\laravel-log-to-db\\src\\Models\\LogToDbCreateObject.php:115)`

I just followed the main tutorial and did this:

`Log::debug("This is an test DEBUG log event");`

All other channels work fine but database one makes the aplication throw 500 Error

**Comment by danielme85** [comment]
Hi, what version of Laravel are you using?

**Comment by JuanAdamuz** [comment]
Last one , 6

**Comment by danielme85** [comment]
Hmm... that was the one change I had to make for Laravel 6 support (I though).
Let me try to fix this code a little, I think this might be an easy fix to make this work across Laravel 5.6 to 6.*.
Would you be able to test with the current dev branch?
```
composer require danielme85/laravel-log-to-db:dev-dev --prefer-source
```

Seems to work fine on the current Laravel 6 project I'm working on, I have to test for older Laravel apps as well.

**Comment by danielme85** [comment]
This should be fixed with #12 
dev-master and latest release would (hopefully) work with Laravel 5 and 6 now.
